### DISCUSSION TITLE
How to use TCP write/recive between different go?

### DISCUSSION THREAD

**Discussion by jiuker** [vlang/v#19359] [Questions and Answers]
**V version:** 
**OS:** 
V 0.2.2 e9d7ff7
**What did you do?**
When I want to use TCP，I cant write TCP write/recive between different go?so How to do?

**What did you expect to see?**
A sample
 
**What did you see instead?**
Shared Object will block?

**Comment by crthpl** [comment]
Can you show your code? Also, a deadlock was fixed a day ago, so please run `v up`.
In addition, using TCP just to communicate between threads (what you call 'go's) is not ideal, use channels if that is what you were trying to achieve.

**Comment by nedpals** [comment]
Please use the appropriate format and tag for posting questions.
